syncd/scripts/syncd_init_common.sh

```diff
@@ -26,10 +26,8 @@ else
     CMD_ARGS=
 fi
 
-# Use temporary view between init and apply except when in fast-reboot
-if [[ "$(cat /proc/cmdline)" != *"SONIC_BOOT_TYPE=fast-reboot"* ]]; then
```

Comment:
We can still keep the check but change the condition. There is a system wide fast-reboot flag that is now used.
May be using that flag is better than 1) removing the check 2) using cmdline which does not change.

Besides, in the current change it is not just fixing the problem with config reload (after fast-reboot). This PR will change the behavior of the fast-reboot as well.